RealEstate/apps/core/views.py

```diff
@@ -219,3 +222,138 @@ def _permission_check(self, request, role, *args, **kwargs):
 
     def get(self, request, *args, **kwargs):
         return render(request, self.template_name, {})
+
+
+class CategoryView(BaseView):
+    """
+    View for the Category Ranking Page.
+    """
+    template_name = 'core/categories.html'
+
+    def _permission_check(self, request, role, *args, **kwargs):
+        return True
```

Comment:
The permission check function is only needed if the implementation is different than the BaseView.